Program.cs

```diff
@@ -200,6 +200,7 @@ private static IEnumerable<Type> GetModelsFromMethod(MethodInfo arg)
 
 		private static IEnumerable<Type> GetModelTypes(Type t)
 		{
+			if (t.GetCustomAttributes().Any(x => x.GetType().Name == "TypeScripterIgnoreAttribute")) yield break;
```

Comment:
This is OK for now, but I'd like to support non-model base classes better in the future.